test/integration/acceptance/custom/headless/headless.go

```diff
@@ -0,0 +1,390 @@
+package headless
```

Comment:
It would be nice to have a general description of the test as the package's documentation

Reply:
brief description added